### ISSUE TITLE
Avoid Crossing Perimeters

### ISSUE THREAD

**Issue by avalero** [daid/LegacyCura#499]
Is there any way to make Cura avoid crossing perimeters when passing from one thing to another? I think previous versions did this, but not anymore

**Comment by daid** [comment]
You mean like combing? Or something else? As if you go from 1 "island" to the next you always have to cross boundaries...

**Comment by avalero** [comment]
Yes I understand that. I mean when I pass from one thing (part) to other
(assuming that I am printing more than 1 STL files) avoid passing over
them. That is, travelling over free space (whenever is possible). This
would be useful as sometimes parts do not have the top layer perfectly and
the nozzle can hit them.

**Comment by daid** [comment]
Ah, yes, "inverted comb" I would call it for now. Should be doable.

**Comment by nallath** [comment]
@daid; Is this implemented?

**Comment by daid** [comment]
not yet, but it's on my list of "I do want to implement this"

**Comment by BagelOrb** [comment]
@daid: is this implemented?
